src/main/java/com/scalar/db/storage/cosmos/Cosmos.java

```diff
@@ -53,7 +53,7 @@ public Cosmos(DatabaseConfig config) {
     this.client =
         new CosmosClientBuilder()
             .endpoint(config.getContactPoints().get(0))
-            .key(config.getPassword())
+            .key(config.getPassword().orElse(null))
```

Comment:
The `key` method throws NullPointerException with an error message ("'key' cannot be null.") when the `password` is EMPTY. I thought it was okay because we basically delegate the configuration parameter check to the db client side (in this case `CosmosClientBuilder`). An invalid password (including null) will cause something error on CosmosClient side.

Reply:
Thank you! Ok, if it is delegating the check to the builder, then I think it is fine. (I want the builder to throw IllegalArgumentException though :( )